zrml/styx/src/lib.rs

```diff
@@ -0,0 +1,105 @@
+#![cfg_attr(not(feature = "std"), no_std)]
+
+extern crate alloc;
+
```

Comment:
Suggestion: Nice to have a README or some DOC here in lib file like other pallets have.

Reply:
Great point. @yornaath Can you add docs that describe the pallet's use?

Reply:
> Suggestion: Nice to have a README or some DOC here in lib file like other pallets have.

Is the standard to have the docs as a top level comment in the lib.rs file with markdown syntax?

Reply:
That's not uncommon, but the prevailing style in our repository is to use doc strings.